csharp/test/Microsoft.ML.OnnxRuntime.Tests.NetCoreApp/InferenceTest.netcore.cs

```diff
@@ -928,51 +846,52 @@ private void TestRegisterCustomOpLibraryV2()
                     throw new Exception(msg + "\n" + ex.StackTrace);
                 }
 
-                ValidateModelWithCustomOps(option);
-
-                // Safe to unload the custom op shared library now
-                UnloadLibrary(libraryHandle);
-            }
-        }
 
-        // test registration by the alternative function name in the custom ops library.
-        // We use DllImport and Marshal.Prelink to make sure the library is loaded and
-        // the symbol is available.
-        internal static class CustomOpLibrary
-        {
-            internal const string ExtensionsDllName = "custom_op_library";
+                using (var session = new InferenceSession(modelPath, option))
+                {
+                    var inputContainer = new List<NamedOnnxValue>();
+                    inputContainer.Add(NamedOnnxValue.CreateFromTensor<float>("input_1",
+                        new DenseTensor<float>(
+                            new float[]
+                            {
+                                1.1f,   2.2f,   3.3f,   4.4f,   5.5f,
+                                6.6f,   7.7f,   8.8f,   9.9f,   10.0f,
+                                11.1f,  12.2f,  13.3f,  14.4f,  15.5f
+                            },
+                            new int[] { 3, 5 }
+                            )));
 
-            [DllImport(ExtensionsDllName, CharSet = CharSet.Ansi,
-                       CallingConvention = CallingConvention.Winapi)]
-            public static extern IntPtr /* OrtStatus* */ RegisterCustomOpsAltName(
-                IntPtr /* OrtSessionOptions* */ sessionOptions,
-                ref OrtApiBase /* OrtApiBase* */ ortApiBase);
-        }
+                    inputContainer.Add(NamedOnnxValue.CreateFromTensor<float>("input_2",
+                        new DenseTensor<float>(
+                            new float[]
+                            {
+                                15.5f,   14.4f,   13.3f,   12.2f,   11.1f,
+                                10.0f,   9.9f,    8.8f,    7.7f,    6.6f,
+                                5.5f,    4.4f,    3.3f,    2.2f,    1.1f
+                            },
+                            new int[] { 3, 5 }
+                            )));
 
-        [SkipNonPackageTests(DisplayName = "TestRegisterCustomOpsWithFunction")]
-        private void TestRegisterCustomOpsWithFunction()
-        {
-            using (var option = new SessionOptions())
-            {
-                try
-                {
-                    Marshal.Prelink(typeof(CustomOpLibrary).GetMethod("RegisterCustomOpsAltName"));
```

Comment:
For unknown reason, this "RegisterCustomOpsAltName " method cannot be found.

Reply:
The symbol should be in the custom ops library but possibly Marshal.Prelink works differently on Linux. The test isn't essential as we're calling RegisterCustomOps directly for extensions so it's a different path.